readme.md

```diff
@@ -7,7 +7,8 @@ Known Issues
 - Heart hitbox is probably not accurate.
 - On the sans_platforms4 and sans_platforms4hard attacks, the platform is supposed to accelerate from 0 to its full speed, but I was lazy and started it at full speed immediately. This is only a problem if you try to dodge the bones without jumping.
 - Sans dialog is missing. I may add something at some future date, but I have no intention of putting the original text in, as it really only makes sense in the context of the whole game.
-- Missing documentation for custom attacks. I intend to add this at some point, but for now, you can reference the *.csv files in the repository. I may change how this works later, so don't be surprised if your custom attack breaks.
+
+Documentation for [custom attacks](Documentation/README.MD)
```

Comment:
I think this would be more appropriate in the main 'Bad Time Simulator (Sans Fight)' section rather than tacked on to the end of 'Known Issues'